src/api/app/models/scm_webhook.rb

```diff
@@ -0,0 +1,34 @@
+# Contains the payload extracted from a SCM webhook and provides helper methods to know which webhook event we're dealing with
+class ScmWebhook
```

Comment:
couldn't we call it `SCMWebhook` ?

Reply:
I decided to go with _Scm_ in lowercase to be consistent with other classes also using this case, like `ReportToScmJob`, `TriggerControllerService::ScmExtractor`, `BadScmHeaders` and `BadScmPayload`.